qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -777,6 +781,35 @@ def _on_url_changed(self, tab, url):
         if not self.widget.page_title(idx):
             self.widget.set_page_title(idx, url.toDisplayString())
 
+    def _mode_override_on_tab_changed(self, tab):
+        """Override mode if url matches pattern on tab changed.
+
+        Args:
+            tab: The WebView where the title was changed.
+        """
+        url = tab.url()
+        self._mode_override(url)
```

Comment:
I suppose this could be a lambda where the signals are connected.

Reply:
Done